template/Project/FeatherHttpProject.csproj

```diff
@@ -9,7 +9,7 @@
     <TargetFramework>netcoreapp3.1</TargetFramework>    
     <IncludeContentInPack>true</IncludeContentInPack>
     <IncludeBuildOutput>false</IncludeBuildOutput>
-    <ContentTargetFolders></ContentTargetFolders>    
+    <ContentTargetFolders>content/content</ContentTargetFolders>    
```

Comment:
I've added content/content here to try to address an issue where when applying the installed template it goes into a root folder of `netcoreapp31` 
My testing indicated that adding this line here changed `netcoreapp31` to `content` which is perhaps marginally better but not ideal. I've tried multiple experiments and tests to try to make this behave the way I think it should, i.e. if you `dotnet new featherhttp -o .` it should add the project files into the directory instead of in a nested directory. 
Currently, this is not behaving the way I expect it to but I've been unable to resolve it. 
I've tried:

* modifying the output directory on build
* omitting any value in ContentTargetFolders (results in `netcoreapp31` as the nested directory)
* using powershell / build tasks to organize project assets similar to the tutorial implementation.

and a few more variations on the above
I've consulted the following resources to seek guidance on fixing this problem:

* https://docs.microsoft.com/en-us/dotnet/core/tutorials/cli-templates-create-template-pack
* https://docs.microsoft.com/en-us/nuget/reference/msbuild-targets
* https://github.com/dotnet/dotnet-template-samples
* https://devblogs.microsoft.com/dotnet/how-to-create-your-own-templates-for-dotnet-new/
* https://github.com/petabridge/petabridge-dotnet-new/tree/dev/src/templates //lots of ideas from this on how to add options for this template later
* https://github.com/dotnet/templating/wiki/Available-templates-for-dotnet-new //looked at several of these but not all of them - could not find guidance
* https://docs.microsoft.com/en-us/dotnet/core/tools/csproj

Do you know where I can look further to solve this problem?